TypeScriptFrontend/src/game/scenes/main.ts

```diff
@@ -821,27 +824,29 @@ export class MainScene extends Scene {
                                 }
                             );
                             capitalText.setOrigin(0, 1);  // Align to bottom-left
-                    
-                            const capitalNumber = this.add.text(
-                                capitalText.x + capitalText.width,
-                                y - 15,
-                                `${capitalCount}`,
-                                {
-                                    fontFamily: "Arial",
-                                    fontSize: "23px",
-                                    color: '#000000', // Black color for the number
-                                }
-                            );
-                            capitalNumber.setOrigin(0, 1);
 
-                            this.capitalTexts.push(capitalText, capitalNumber);
+                            if (player instanceof MyCreditPlayer) {
```

Comment:
I was thinking this display logic would also be moved out of main. For example OtherPlayer would have display logic for name and count, and then credit player would override the function and also display capital count. Does that make sens?

Reply:
makes sense! i'll take a look at that tonight

Reply:
changes made, please review most recent commit!